dependencies/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ add_subdirectory(llvm)
 
 add_subdirectory(jpeg)
 add_subdirectory(png)
+add_subdirectory(spirv)
```

Comment:
This is where `if (TARGET_SPIRV)` should go. Then unindent `spirv/CMakeLists.txt`.